float64 x
float64 y
float64 z
float64 w
float64 t1
float64 t2
float64 t3
int32 objectId
int32 cameraId
